tests/ignite/handlers/test_checkpoint.py

```diff
@@ -404,82 +407,74 @@ def _test(to_save, obj, name, score_name=None):
     _test(to_save, {"model": model.state_dict(), "optimizer": optimizer.state_dict()}, "checkpoint", "epoch")
 
 
-def test_checkpoint_with_score_function_and_trainer_epoch():
```

Comment:
This test has only one configuration. No need to write it using `pytest.mark.parametrize`